src/utils/upload/fileInspector.js

```diff
@@ -83,7 +87,9 @@ const inspect10XFile = async (file) => {
   return Verdict.INVALID_FORMAT;
 };
 
-const inspectRhapsodyFile = async (file) => (isGzipped(file));
+const inspectRhapsodyFile = async (file) => await isGzipped(file);
+
+const inspectParseFile = async (file) => await isGzipped(file);
```

Comment:
I added this because it is necessary, but I don't understand why the `isGzipped` check was moved away from `loadAndCompressIfNecessary` which was more self-contained than the current implementation that requires us to store a `compressed` bool in `file`.

Also for all the techs there is an `isGzipped` check, it should be unified if for some reason we don't move it back into `loadAndCompressIfNecessary`